torchvision/ops/misc.py

```diff
@@ -196,27 +251,57 @@ def __init__(
         bias: Optional[bool] = None,
     ) -> None:
 
-        if padding is None:
-            padding = (kernel_size - 1) // 2 * dilation
-        if bias is None:
-            bias = norm_layer is None
-        layers = [
-            torch.nn.Conv3d(
-                in_channels,
-                out_channels,
-                kernel_size,
-                stride,
-                padding,
-                dilation=dilation,
-                groups=groups,
-                bias=bias,
-            )
-        ]
-        if norm_layer is not None:
-            layers.append(norm_layer(out_channels))
-        if activation_layer is not None:
-            params = {} if inplace is None else {"inplace": inplace}
-            layers.append(activation_layer(**params))
-        super().__init__(*layers)
+        layer = torch.nn.Conv3d
+        super().__init__(
+            in_channels,
+            out_channels,
+            layer,
+            kernel_size,
+            stride,
+            padding,
+            groups,
+            norm_layer,
+            activation_layer,
+            dilation,
+            inplace,
+            bias,
+        )
+
+
+class SqueezeExcitation(torch.nn.Module):
```

Comment:
I assume this code is completely unmodified and Github fails to create a proper diff. Could you confirm?

Reply:
Yes, I haven't touched SqueezeExcitation